notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>It seems that the only use of the config file is to get input and output link. We can create just two global variable at the top D3_INPUT_CSV_PREFIX, and D3_OUTPUT_TABLE_PREFIX and use that in the notebook everywhere. The ML model config file can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>that makes sense, also removes some overhead while creating pipelines 👍</p>